src/sass/static/pages.scss

```diff
@@ -470,3 +470,213 @@ body #not_authenticated_financial ul.checked > li {
         margin: 0 auto;
     }
 }
+
+.careers-content {
+    .career {
+        margin-top: 0;
+
+        .top-banner {
+            background: url('../images/pages/careers/careers-1_bg@3x.jpg') no-repeat;
+            background-size: cover;
+            min-height: 300px;
+            position: relative;
+            background-position: center center;
```

Comment:
is it the same styling if we emit one `center`? or we need to keep both?

Reply:
Should be the same if we use only one `center`, i'll update it along with the other fixes.